.ci/pipeline/ci.yml

```diff
@@ -46,7 +46,45 @@ jobs:
       targetPath: '$(release.dir)'
   - script: |
       .ci/scripts/test.sh --test-kind examples --build-dir $(release.dir) --platform $(platform.type)
-    displayName: 'examples'  
+    displayName: 'examples'
+- job: daal4py
+  dependsOn: Linux
+  timeoutInMinutes: 0
+  variables:
+    release.dir: '__release_lnx_gnu'
+  pool:
+    vmImage: 'ubuntu-18.04'
+  strategy:
+    matrix:
+      lnx32e:
+        platform.type : 'lnx32e'
+    maxParallel: 2
+  steps:
+  - task: DownloadPipelineArtifact@2
+    inputs:
+      artifact: '$(platform.type) build'
+  - script: |
+        .ci/env/daal4py-dpcpp.sh
+    displayName: 'create dpcpp enviroment'
+  - script: |
+      sudo apt-get install git
+      git clone https://github.com/IntelPython/daal4py.git
+    displayName: clone daal4py
+  - script: |
+      wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
+      bash miniconda.sh -b -p $HOME/miniconda
+      export PATH="$HOME/miniconda/bin:$PATH"
+      conda create -q -y -n CB python=3.7 conda-build conda-verify
+    displayName: Create Anaconda environment
+  - script: |
+      export DPCPP_VAR=/opt/intel/inteloneapi/compiler/latest/env/vars.sh
+      export DAALROOT='$(platform.type) build'/daal/latest
```

Comment:
it should be something like this  DAALROOT="$(Pipeline.Workspace)/$(platform.type)\ build/__release_lnx_gnu/daal/latest"